src/pages/run-a-node.tsx

```diff
@@ -0,0 +1,911 @@
+  <Flex
+    bg="grayBackground"
+    border="1px"
+    borderColor="#d1d1d1"
+    borderRadius="5px"
+    color="text"
+    py="0"
+    px="8"
+    {...props}
+  />
+)
+
+const BuildBox = (props: ComponentProps<typeof Container>) => (
+  <Container
+    direction="column"
+    bg="preBackground"
+    flex={1}
+    p="8"
+    sx={{
+      "& > p:last-of-type": {
+        mb: 8,
+      },
+      "li:last-child": {
+        mb: 0,
+      },
+    }}
+    {...props}
+  />
+)
+
+const BuildBoxSpace = (props: ChildOnlyProp) => (
+  <BuildBox
+    direction="column"
+    justify="space-between"
+    _hover={{
+      transform: "scale(1.02)",
+      transition: "transform 0.1s",
+    }}
+    {...props}
+  />
+)
+
+const FullyLoaded = (props: ChildOnlyProp) => (
+  <Container
+    direction="column"
+    justify="space-between"
+    lineHeight="200%"
+    flex={1}
+    px="8"
+    py="8"
+    _hover={{
+      transform: "scale(1.02)",
+      transition: "transform 0.1s",
+    }}
+    sx={{
+      p: {
+        fontSize: "110%",
+      },
+      code: {
+        fontWeight: 600,
+        lineHeight: "125%",
+      },
+      "li:last-child": {
+        mb: 0,
+      },
+    }}
+    {...props}
+  />
+)
+
+const SvgTitle = (props: ChildOnlyProp) => (
+  <Flex gap="4" align="center" {...props} />
+)
+
+const ButtonContainer = (props: ChildOnlyProp) => (
+  <Flex
+    gap="4"
+    mt="auto"
+    direction={{ base: "column", lg: "row" }}
+    {...props}
+  />
+)
+
+const BuildContainer = (props: ChildOnlyProp) => (
+  <Container
+    direction="column"
+    py="8"
+    px={{ base: 0, md: 8 }}
+    borderRadius="none"
+    border="none"
+    bg="none"
+    {...props}
+  />
+)
+
+const StakingCalloutContainer = (props: ChildOnlyProp) => (
+  <SplitContent
+    w="full"
+    p="8"
+    bg="linear-gradient(
+      262.78deg,
+      rgba(152, 186, 249, 0.25) 0%,
+      rgba(207, 177, 251, 0.25) 53.12%,
+      rgba(151, 252, 246, 0.25) 100%
+    )"
+    {...props}
+  />
+)
+
+const StrongParagraph = (props: BoxProps) => (
+  <Text fontSize="150%" fontWeight="semibold" {...props} />
+)
+
+const H2 = (props: HeadingProps) => (
+  <OldHeading
+    fontSize={{ base: "2xl", md: "2rem" }}
+    lineHeight={1.4}
+    {...props}
+  />
+)
+
+const H3 = (props: HeadingProps) => (
+  <OldHeading
+    as="h3"
+    fontSize={{ base: "xl", md: "2xl" }}
+    lineHeight={1.4}
+    {...props}
+  />
+)
+
+const H4 = (props: ChildOnlyProp) => (
+  <OldHeading
+    as="h4"
+    fontSize={{ base: "md", md: "xl" }}
+    lineHeight={1.4}
+    fontWeight="medium"
+    {...props}
+  />
+)
+
+const Width60 = (props: ChildOnlyProp) => <Box w="full" flex={3} {...props} />
+
+const Width40 = (props: ChildOnlyProp) => (
+  <Center w="full" flex={2} {...props} />
+)
+
+type RunANodeCard = {
+  image: typeof ChakraIcon
+  title: string
+  preview: ReactNode
+  body: string[]
+  alt: string
+}
+
+export const getStaticProps: GetStaticProps<{}, {}> = async (context) => {
```

Comment:
Use the `satisfies GetStaticProps` from
```suggestion
export const getStaticProps = async (context) => {
```